full/src/main/java/apoc/dv/VRCatalogHandler.java

```diff
@@ -0,0 +1,100 @@
+package apoc.dv;
+
+import apoc.SystemLabels;
+import apoc.SystemPropertyKeys;
+import apoc.util.JsonUtil;
+import apoc.util.Util;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.internal.helpers.collection.Pair;
+import org.neo4j.logging.Log;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class VRCatalogHandler {
```

Comment:
```suggestion
public class DVCatalogHandler {
```
?

Reply:
Changed in `DataVirtualizationCatalogHandler`